### PR TITLE
Order data updating

### PR DESCRIPTION
**Pull request by ilikesymmetry**
Allows querying campus eateries while we fetch from yelp.

### CODE REVIEW

**Review by meganle — APPROVED** [review]
Amazing, nice work!

File: src/data.py [hunk 1]
```diff
@@ -45,9 +45,11 @@ def start_update():
     fill_empty_menus(campus_eateries)
     statics_json = requests.get(STATIC_EATERIES_URL).json()
     parse_static_eateries(statics_json)
+    Data.update_data(campus_eateries)
+    print('[{0}] Updating collegetown'.format(datetime.now()))
```
Comment (young-k) [inline src/data.py]: i don't think you need `{0}` if it's just 1, you can just have `{}`

File: src/data.py [hunk 2]
```diff
@@ -45,9 +45,11 @@ def start_update():
...
     yelp_query = collegetown_search()
     parse_collegetown_eateries(yelp_query)
-    Data.update_data(campus_eateries, collegetown_eateries)
+    Data.update_collegetown(collegetown_eateries)
```
Comment (young-k) [inline src/data.py]: rename to `update_collegetown_data`

File: src/data.py [hunk 3]
```diff
@@ -45,9 +45,11 @@ def start_update():
     fill_empty_menus(campus_eateries)
     statics_json = requests.get(STATIC_EATERIES_URL).json()
```
Comment (young-k) [inline src/data.py]: `static_json`

**Review by young-k — CHANGES_REQUESTED** [review]
Meant to request changes whoops

**Comment by ilikesymmetry** [comment]
changes made @young-k

**Review by young-k — APPROVED** [review]

**Pull request merged**